src/green/troubleshooting/green-reset-failed-what-to-do.md

```diff
@@ -0,0 +1,26 @@
+---
+zendesk:
+  article_id: 28896513100701
+  name: Resetting Home Assistant Green failed. What can I do?
+  position: 5
+  labels: green, troubleshooting
+---
+
+## Symptom
+
+You followed the steps on [resetting the device without SD card (OS version 13.1 and later)](/hc/en-us/articles/25161225495837) but the reset failed.
+
+## Resolution
+
+1. Check which version of Home Assistant Operating System you have installed on Home Assistant Green.
```

Comment:
I thought all of these articles were written with the assumption that the reader was running the current version?

Reply:
ok.